manifests/f/fjsoft/MyPhoneExplorer/1.8.15/fjsoft.MyPhoneExplorer.locale.de.yaml

```
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: fjsoft.MyPhoneExplorer
+PackageVersion: 1.8.15
+PackageLocale: de

```

```suggestion
PackageLocale: de-DE
```
Invalid Locale code.

---

Well, the developer is from Austria, so I am not sure if de-DE or de-AT would be appropriate

---

It does not matter where the Developer is from, the only thing that matters is what language the Metadata is in. Aka your ShortDescription/Description. And Homepage, License, LicenseUrl, PublisherUrl, PublisherSupportUrl, Copyright and CopyrightUrl if that gets added.